src/platform/forms-system/src/js/review/SubmitButtons.jsx

```diff
@@ -24,11 +24,13 @@ export default function SubmitButtons(props) {
   } = props;
 
   const appType = formConfig?.customText?.appType || APP_TYPE_DEFAULT;
+  const buttonText =
+    formConfig.customText?.submitButtonText || `Submit ${appType}`;
```

Comment:
Should we add this new `formConfig` property to the documentation anywhere with an example/explanation of what it does?

Reply:
Good call. Yes. Yes, we should. It's in another repo, though, so that'll be another PR. I'll put it on my to-do list so I don't forget.